content/404.md

```diff
@@ -4,6 +4,6 @@ title: Page Not Found
 permalink: 404.html
 ---
 
-We couldn't find what you were looking for.
+તમે જે શોધી રહ્યા હતા તે અમે શોધી શક્યા નહીં.
 
-Please contact the owner of the site that linked you to the original URL and let them know their link is broken.
+કૃપા કરી તે સાઇટના માલિકનો સંપર્ક કરો કે જેણે તમને મૂળ URL સાથે લિંક કરી દીધો છે અને તેમને જણાવો કે તેમની લિંક તૂટી ગઈ છે.
```

Comment:
I think this should be 
"કૃપા કરી તમને જે વેબસાઈટે મૂળ URL સાથે લિંક કર્યા એના મલિકનો સંપર્ક કરો અને કહો કે આ તૂટેલી લિંક છે."